### ISSUE TITLE
build in NUMBER function 

### ISSUE THREAD

**Issue by LokiMidgard** [blushingpenguin/Fluent.Net#6]
If I've seen it correctly the current implementation of the number function does nothing.

What are your plans for the future?
 - Do you intent to provided the same parameter that the reference implementation has? 
 - Do you will provide the options c# offers for formating?
 - Just keep this as a placeholder so messages copied to c# will not throw errors (but won't get treated differently)

**Comment by blushingpenguin** [comment]
> Do you intent to provided the same parameter that the reference implementation has?
Yes, eventually that was the intention -- we are using fluent and the same set of translations for everything, so the c# version on the backend and the JS version on the client side. Currently we don't have any number formatting requirements though so I haven't implemented it yet.

> Do you will provide the options c# offers for formating?
It would be the options the JS implementation provides (Intl.NumberFormat) -- I need to find, port or create a c# implementation of that.

> Just keep this as a placeholder so messages copied to c# will not throw errors (but won't get treated differently)
If you wanted to do that, you could just add a different implementation to MessageContext.Functions -- supplied function implementations will be used in preference to the built in ones.
